ihmc-robotics-toolkit/build.gradle.kts

```diff
@@ -29,10 +29,10 @@ mainDependencies {
    api("us.ihmc:ihmc-commons:0.35.0")
    api("us.ihmc:ihmc-messager-kryo:0.2.0")
    api("us.ihmc:ihmc-matrix-library:0.19.0")
-   api("us.ihmc:ihmc-graphics-description:0.25.1")
+   api("us.ihmc:ihmc-graphics-description:0.21.1")
```

Comment:
This doesn't make sense.  You will eventually run into problems when incrementing the version.